src/components/sessions/Session.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+import PropTypes from 'prop-types'
+import { Button, Grid, Segment } from 'semantic-ui-react'
+import moment from 'moment'
+import QuestionInSession from './QuestionInSession'
+
+import withCSS from '../../lib/withCSS'
+
+const Session = ({ createdAt, head, id, intl, name, questions, status }) => {
+  let buttonContent = ''
+  let buttonIcon = ''
+  switch (status) {
```

Comment:
We can probably simplify this to not need a switch statement. Will try to find another way.